cmd/package.go

```diff
@@ -25,7 +26,10 @@ import (
 )
 
 func InitPackage(packagePath string, p *core.Package) error {
-	// We have to create hte package directory and it's metadata directory.
+	// Remember when the package was initialized.
+	p.Created = time.Now().Format("2006-01-02 15:04")
```

Comment:
May I suggest the use of standard datetime format? For example, `time.Now().Format(time.RFC3339)` would print something like `2017-09-11T10:30:50+02:00`. This will probably break some things, but we can easily regenerate the new packages prior to the next Capstan release.

Reply:
Done. We're now marshalling into RFC3339 while unmarshalling from either RFC3339 either our user friendly format, so we're backwards compatible.